engine/table/src/test/java/io/deephaven/engine/table/impl/TestPartitionBy.java

```diff
@@ -181,16 +181,19 @@ public void testNewKeysAfterResultReleased() {
 
         try (final SafeCloseable ignored1 = LivenessScopeStack.open(subTablesScope, true)) {
 
-            final TableMap byKey;
+            final PartitionedTable byKey;
             final Table tableA;
             final Table tableB;
 
+            final LivenessScope subTableManager = new LivenessScope();
             try (final SafeCloseable ignored2 = LivenessScopeStack.open()) {
                 byKey = table.partitionBy("Key");
-                try (final SafeCloseable ignored3 = LivenessScopeStack.open(subTablesScope, false)) {
-                    tableA = byKey.get("A");
-                    tableB = byKey.get("B");
-                }
+                tableA = getPartition(byKey, "A");
```

Comment:
That is a good question. I *think* I proposed adding a `get` to `PartitionedTable`, and maybe you weren't a fan of that idea? I could use some input, here. I don't think there are *tons* of use cases where users need to `get` from one of these, other than the well-known GUI-subscription case, and most of the time that it *is* necessary is because someone is misusing a `TableMap` in lieu of a `Map`.

Reply:
I will promote and document the `static` methods in `TstUtils` to be members of the `PartitionedTable` interface.